main.tf

```diff
@@ -114,7 +117,7 @@ resource "ibm_database" "rabbitmq_database" {
 
   ## This block is for if host_flavor IS NOT set
   dynamic "group" {
-    for_each = local.host_flavor_set ? [] : [1]
+    for_each = local.host_flavor_set && var.backup_crn == null ? [] : [1]
```

Comment:
That logic was wrong and later improved in postgresql to:

```icl
for_each = local.host_flavor_set == false && !local.recovery_mode ? [1] : []
```

Without pitr I think it needs to be 

```hcl
for_each = local.host_flavor_set == false && !var.backup_crn ? [1] : []
```